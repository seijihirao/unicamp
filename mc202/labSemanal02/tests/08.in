42
42
1 3 5 7 9 11 13 15 17 19 21 23 25 27 29 31 33 35 37 39 41 42 40 38 36 34 32 30 28 26 24 22 20 18 16 14 12 10 8 6 4 2
