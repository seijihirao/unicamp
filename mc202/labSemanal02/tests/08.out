1323 924 1323
